Mage.Server/src/main/java/mage/server/Session.java

```diff
@@ -371,12 +371,14 @@ public void fireCallback(final ClientCallback call) {
             call.setMessageId(messageId++);
             callbackHandler.handleCallbackOneway(new Callback(call));
         } catch (HandleCallbackException ex) {
-            User user = UserManager.getInstance().getUser(userId);
-            logger.warn("SESSION CALLBACK EXCEPTION - " + (user != null ? user.getName() : "") + " userId " + userId);
-            logger.warn(" - method: " + call.getMethod());
-            logger.warn(" - cause: " + getBasicCause(ex).toString());
-            logger.trace("Stack trace:", ex);
-            userLostConnection();
+            ex.printStackTrace();
+            UserManager.getInstance().getUser(userId).ifPresent(user-> {
+                logger.warn("SESSION CALLBACK EXCEPTION - " + (user != null ? user.getName() : "") + " userId " + userId);
```

Comment:
this is executed only now, if a user is present, so user check != null makes no longer sense.